samples/net/wifi/test_certs/install_certs.sh

```diff
@@ -0,0 +1,108 @@
+#   - client.pem: Client certificate for Wi-Fi Enterprise mode (Phase 1)
+#   - server.pem: Server certificate for Wi-Fi Enterprise mode, for used in AP mode.
+#   - ca2.pem: CA certificate for Wi-Fi Enterprise mode (Phase 2)
+#   - client-key2.pem: Client private key for Wi-Fi Enterprise mode (Phase 2)
+#   - client2.pem: Client certificate for Wi-Fi Enterprise mode (Phase 2)
+#
+# Each certificate file is associated with a specific security tag (sec_tag) which is used during
+# installation.
+#
+# The script performs the following steps:
+#   1. Checks if the required arguments are provided.
+#   2. Validates the existence of the certificate directory.
+#   3. Checks if the `device_credentials_installer` tool is available.
+#   4. Iterates over the expected certificate files and installs them to the device if they exist.
+#   5. Logs the success or failure of each certificate installation.
+#
+# Note:
+#   - If a certificate file is missing, the script will skip its installation and log a warning.
+#   - The script will terminate on the first encountered error (set -e).
+set -e
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 <path_to_certs> [port] [mode]"
+    exit 1
+fi
+
+CERT_PATH=$1
+PORT=${2:-/dev/ttyACM1}  # Default port is /dev/ttyACM1 if not provided
+MODE=${3:-STA}  # Default mode is STA if not provided
+
+if [ ! -d "$CERT_PATH" ]; then
+    echo "Error: Directory $CERT_PATH does not exist."
+    exit 1
+fi
+
+if ! command -v device_credentials_installer &> /dev/null; then
+    echo "Error: device_credentials_installer could not be found."
+    echo "Please install it using: pip3 install nrfcloud-utils"
+    exit 1
+fi
+
+# Should match the device side tags
+WIFI_CERT_SEC_TAG_BASE=0x1020001
+declare -A WIFI_CERT_SEC_TAG_MAP=(
+    ["ca.pem"]=$WIFI_CERT_SEC_TAG_BASE
+    ["client-key.pem"]=$(($WIFI_CERT_SEC_TAG_BASE + 1))
+    ["server-key.pem"]=$(($WIFI_CERT_SEC_TAG_BASE + 2))
+    ["client.pem"]=$(($WIFI_CERT_SEC_TAG_BASE + 3))
+    ["server.pem"]=$(($WIFI_CERT_SEC_TAG_BASE + 4))
+    ["ca2.pem"]=$(($WIFI_CERT_SEC_TAG_BASE + 5))
+    ["client-key2.pem"]=$(($WIFI_CERT_SEC_TAG_BASE + 6))
+    ["client2.pem"]=$(($WIFI_CERT_SEC_TAG_BASE + 7))
+)
+
+# Select certificates based on mode
+if [ "$MODE" == "AP" ]; then
+    CERT_FILES=("ca.pem" "server-key.pem" "server.pem")
+else
+    CERT_FILES=("ca.pem" "client-key.pem" "client.pem" "ca2.pem" "client-key2.pem" "client2.pem")
+fi
+
+total_certs=${#CERT_FILES[@]}
+processed_certs=0
+
+for cert in "${CERT_FILES[@]}"; do
+    processed_certs=$((processed_certs + 1))
+    echo "Processing certificate $processed_certs of $total_certs: $cert"
+
+    if [ ! -f "$CERT_PATH/$cert" ]; then
+        echo "Warning: Certificate file $CERT_PATH/$cert does not exist. Skipping..."
+        continue
+    fi
+
+    sec_tag=${WIFI_CERT_SEC_TAG_MAP[$cert]}
+    sec_tag_int=$((sec_tag))
+    ret=$(device_credentials_installer --local-cert-file "$CERT_PATH/$cert" --cmd-type tls_cred_shell \
```

Comment:
This depends on https://github.com/nRFCloud/utils/pull/55 and might not work on non-nRF devices, so, I might move this script to a `nordic` vendor location and make it nRF specific or come up with a generic script, to be discussed.

Reply:
Tested with a STM32 Nucelo board and seems to work.